### ISSUE TITLE
Add social login 

### ISSUE THREAD

**Issue by akmLaVx** [AdguardTeam/AdguardForiOS#1507]
### Issue Details

- **AdGuard version:**
  - 3.1.4
- **Device model and storage size:**
  - unspecified
- **Operating system and version:**
  - unspecified

### Expected Behavior

Login form under AdGuard > About > License > Login should have a Google login widget

### Actual Behavior

Google login widget is missing in app

### Screenshots

<details><summary>Screenshot:</summary>

![image0](https://user-images.githubusercontent.com/33634917/75177743-6b1f7000-573f-11ea-96f4-ee5da8461afa.png)

</details>

### Additional Information

ID: 2564511

**Comment by ivgotfri** [comment]
Issue Details
AdGuard version:
3.1.4(254)

Device model and storage size:
Apple iPhone 11 Pro 

Operating system and version:
iOS 13.3.1

Expected Behavior
Login form under AdGuard > About > License > Login should have a Google login widget, but there's no G button on iPhone AdGuard app. (attached; iphone adguard)
However, on the web page of AdGuard, I can log in easily by clicking G button on log in page. (attached: desktop adguard)

Actual Behavior
Google login widget is missing in iOS app

Screenshots
![iphone adguard](https://user-images.githubusercontent.com/61449399/75223060-015ba080-57e9-11ea-83e2-0110d565a13f.png)
![desktop adguard](https://user-images.githubusercontent.com/61449399/75223072-04ef2780-57e9-11ea-83b1-d671f82a5f33.png)

**Comment by zznosar** [comment]
Should this be a feature request? If i remember correctly, Google Widget was never in the Log In form since v 3.0.0 when they introduced the ios mobile license.

The web online form is different from the ios log in form.

**Comment by ivgotfri** [comment]
Thanks for your reply, @zznosar 

I'm not a developer at all and I purchased AdGuard family pack for lifetime recently. When I use this app, I need to make an AdGuard account. So, I chose my Google account for convenience by just clikcing G button under the log in form on AdGuard web site.
Now with my iPhond AdGuard app, there's no G button for easy login with my google account.
I didn't know how to solve this problem, then I sent an e-mail to AdGuard tech support. They made this thread.

I'm waiting for AdGuard reply.

**Comment by ameshkov** [comment]
Temporary solution - log in just using the same email address you have in your Google account (you may need to set a password in your AdGuard account for that).

We don't have social login in the iOS app yet so assigning this on v4.1.

**Comment by ivgotfri** [comment]
Thanks for the good suggestion, @ameshkov 
That's what I thought to solve this problem.

First, I tried to log in with my Google ID & password with a desktop web browser by putting them in the login fields, not by clicking the G button at the bottom. But I failed to log in.
![error](https://user-images.githubusercontent.com/61449399/75302129-441c8780-5880-11ea-8153-22ef7a09f6e0.png)

Second, I tried to make an AdGuard account with my Gmail address which is used to log in with the G button, but I failed.
![error1](https://user-images.githubusercontent.com/61449399/75302210-74642600-5880-11ea-88f7-33f3f0196b6c.png)

Third, I tried to set a new password after I log in by clicking G button, then log in again with the e-mail address and the new password, but I failed to set a new one.
![error2](https://user-images.githubusercontent.com/61449399/75302281-a1183d80-5880-11ea-948a-b3a48019482b.png)

I can click the G button to log in on my Mac laptop and Windows desktop computer, but I can't log in on my iPhone. How can I solve this problem?
